stock_manual_transfer/security/ir.model.access.csv

```diff
@@ -1,3 +1,3 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
-access_stock_can_stock_manual_transfer,stock_can_stock_manual_transfer,model_stock_manual_transfer,group_stock_manual_transfer,1,1,1,1
```

Comment:
Does changing the ID and name of the access right require updating the records in the ir_model_data and ir_model_access tables?

Additionally, it seems this change is missing from the commit description.

Reply:
@CLaurelB 

> Does changing the ID and name of the access right require updating the records in the ir_model_data and ir_model_access tables?

No, because these are updatable records (`noupdate=0`). It would be similar to update views.

> Additionally, it seems this change is missing from the commit description.

I didn't include it because I didn't consider it as necessary, but done, "better explicit than implicit".